package.json

```
@@ -25,7 +25,8 @@
     "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
   ],
   "files": [
-    "index.js"
+    "index.js",
+    "light.js"
```

```suggestion
    "core.js",
    "index.js",
    "light.js"
```

All files that are to be published must be listed here, but otherwise there’s no way to specify two entries. So this is fine!